pkgs/by-name/wi/wio/package.nix

```diff
@@ -0,0 +1,76 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, alacritty
+, cage
+, cairo
+, libxkbcommon
+, makeWrapper
+, mesa
+, meson
+, ninja
+, pkg-config
+, udev
+, unstableGitUpdater
+, wayland
+, wayland-protocols
+, wlroots
```

Comment:
should use `wlroots_0_16`

https://github.com/NixOS/nixpkgs/pull/269359  will break it

Reply:
Good to know. I will open a PR for this later.